Tests/Functional/ControllerResolverTest.php

```diff
@@ -67,4 +69,19 @@ public function testAnnotationControllerServiceExtendingClassicService()
 
         $this->assertEquals('hello', $client->getResponse()->getContent());
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testInvokableControllerAsService()
+    {
+        if (version_compare(Kernel::VERSION_ID, '20600') < 0) {
```

Comment:
Because change that allows to use invokable controllers was introduced in 2.6 
https://github.com/symfony/symfony/blob/2.6/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerResolver.php#L66-L67 so it will not work for `sf < 2.6`